services/src/main/java/org/exoplatform/officeonline/WOPIService.java

```diff
@@ -1121,11 +1124,11 @@ protected void addFileURLProperties(Map<String, Serializable> map, Node node, Ed
       }
     }
 
-    String explorerLink = symlink != null ? explorerLink(symlink.getPath()) : explorerLink(node.getPath());
-    URI explorerUri = explorerUri(config.getBaseUrl(), explorerLink);
-    if (explorerUri != null) {
-      map.put(CLOSE_URL, explorerUri.toString());
-      map.put(FILE_VERSION_URL, explorerUri.toString() + "&versions=true");
+    Node explorerNode = symlink != null ? symlink : node;
+    String explorerUrl = getExplorerURL(explorerNode, config.getBaseUrl());
+    if (explorerUrl != null) {
+      map.put(CLOSE_URL, explorerUrl);
+      map.put(FILE_VERSION_URL, explorerUrl + "&versions=true");
```

Comment:
StringBuilder will be more efficient if it's often called op.

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/28/commits/09dd43c4c6497a4cd87cc727a522b7c45c3eefdd